deployment/Compose/compose/.env.example

```
@@ -27,10 +27,10 @@ KEYCLOAK_ADMIN_PASSWORD=oqm_admin
 
 # Core API
 CORE_API_PORT=8101
-CORE_API_PORT_IMAGE_TAG=4.4.1
+CORE_API_PORT_IMAGE_TAG=6.0.0
 
 # Base Station
 CORE_BASE_STATION_PORT=8102
-CORE_BASE_STATION_IMAGE_TAG=1.14.1
+CORE_BASE_STATION_IMAGE_TAG=1.19.1-SNAPSHOT
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The CORE_BASE_STATION_IMAGE_TAG key should go before the CORE_BASE_STATION_PORT key

(UnorderedKey)